File: 이동령/Programmers/Level 3/최고의 집합.js
function solution(n, s) {
    let arithmeticMean = Math.floor(s / n);
    let remainder = s % n;
    if(arithmeticMean === 0) return [-1];
    else return Array.from({length: n}, (_, i) => {
        return i < n - remainder ? arithmeticMean : arithmeticMean + 1
    });
}

File: 이승연/Programmers/Level 3/최고의_집합.java
public class 최고의_집합 {
    class Solution {
        public int[] solution(int n, int s) {
            int[] answer = new int[n];
            if (n > s) return new int[]{-1};
            int r = s%n;
            for(int i=0; i<answer.length; i++) {
                answer[i] = s/n;
            }

            for(int i=0; i<r; i++) {
                answer[answer.length-1-i] += 1;
            }
            return answer;
        }
    }
}